.lando.yml

```diff
@@ -29,13 +29,16 @@ services:
       - if [ ! -e $LANDO_MOUNT/wp-cli.yml ]; then cp $LANDO_MOUNT/config/wp-cli.yml $LANDO_MOUNT/wp-cli.yml; fi
     run:
       # Setup WordPress for contributing via git.
+      # TODO: Rename core-git to core or core-dev.
```

Comment:
Let's go with `core-dev` as this indicates that we are in a development version. We might wanna include a `core` directory at a later stage so that people can easily run from a packaged version if they don't care about contributing to core.